.ci-env/flags.sh-e

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+# This flag indicates if the tag should be treated as a latest one, which is going to trigger
+# - npm publish with `latest` tag, which will become new default version
+# - main version incrementation
+latest=false
```

Comment:
Should there be new file named like that?

Reply:
this is autogenerated

Reply:
this was autogenerated